libunsf.c

```diff
@@ -1082,7 +1069,7 @@ static int unsf_mkdir(char *dir) {
 static void make_directories(UnSF_Options *options, SampleBank *sample_bank) {
     int i, tonebank_count = 0;
     char tmpname[80];
-    char *directory = NULL;
+    char directory[1024];
```

Comment:
On 9/8/16, Bret Curtis <notifications@github.com> wrote:
>> @@ -1082,7 +1069,7 @@ static int unsf_mkdir(char *dir) {
>>  static void make_directories(UnSF_Options *options, SampleBank
>> *sample_bank) {
>>      int i, tonebank_count = 0;
>>      char tmpname[80];
>> -    char *directory = NULL;
>> +    char directory[1024];
>
> Do we really want a path character limit of 1024? I wanted to move away from
> arbitrary boundaries. :/

I know that it's not favorable, but it eases things.
[PATH_MAX] or an equivalent is an alternative, which
is 4096 for linux. (And I know that many uses for this
library are probably under 128.)

Or, memory management can be tweaked so that it
doesn't rape the heap